### PR TITLE
Hotfix: Add windows trust store switch

### PR DESCRIPTION
**Pull request by alexanderkasten**
## Changes

1. Update Atlas Engine Version
2. Add TLS Use Windows Trust Store switch on Windows

## Issues

PR: #2128 

## How to test the changes

> Describe how others can test your changes (not required when fixing typos, linter errors and such)

### CODE REVIEW

File: src/modules/config-panel/config-panel.html [hunk 1]
```diff
@@ -25,19 +31,43 @@ <h1>Settings</h1>
       </div>
       <div class="card-body">
         <div class="form-check">
-          <label class="form-check-label"><input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates">Trust unauthorized certificates</label>
+          <label class="form-check-label"
+            ><input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates" />Trust
+            unauthorized certificates</label
+          >
```
Comment (ElRaptorus) [inline src/modules/config-panel/config-panel.html:37]: Gibts fürs BPMN Studio eigentlich auch ne prettier config?

```suggestion
          <label class="form-check-label">
            <input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates" />
            Trust unauthorized certificates
          </label>
```

File: src/modules/config-panel/config-panel.html [hunk 2]
```diff
@@ -25,19 +31,43 @@ <h1>Settings</h1>
...
+    <div if.bind="showWindowsTrustStoreSwitch" class="config-panel__card card ">
+      <div class="card-header">
+        TLS
+      </div>
+      <div class="card-body">
+        <div class="form-check">
+          <label class="form-check-label"
+            ><input type="checkbox" class="form-check-input" checked.bind="useWindowsTrustStore" />Use Windows Trust
+            Store</label
+          >
```
Comment (ElRaptorus) [inline src/modules/config-panel/config-panel.html:50]:
```suggestion
          <label class="form-check-label">
            <input type="checkbox" class="form-check-input" checked.bind="useWindowsTrustStore" />
            Use Windows Trust Store
          </label>
```

**Review by ElRaptorus — APPROVED** [review]

File: src/modules/config-panel/config-panel.html (hunk 1 above)
Comment (alexanderkasten) [reply src/modules/config-panel/config-panel.html:37]: Ja :D die hat das so eingestellt 😂

Comment (ElRaptorus) [reply src/modules/config-panel/config-panel.html]: ...
:bomb:

**Pull request merged**